objectRecognitionSource/README.md

```diff
@@ -44,21 +44,90 @@ Additionally, an example VANTIQ project named *objRecExample.zip* can be found i
-    *   [CameraRetriever](#camRet) -- Retrieves images from a directly connected camera using OpenCV.
+    *   [CameraRetriever](#camRet) -- Retrieves images from a directly connected camera using OpenCV<sup>&trade;</sup>.
     *   [NetworkStreamRetriever](#netRet) -- Retrieves images from an IP camera.
     *   [FileRetriever](#fileRet) -- Retrieves images and videos from disk.
     *   [FtpRetriever](#ftpRet) -- Retrieves images through FTP, FTPS, and SFTP.
 
 ## How to Run the Program<a name="objRecMain" id="objRecMain"></a>
 
-1.  If you intend to use any of the implementations that require OpenCV, you must install OpenCV version 4.1.0 for
-Java. The [official site](https://opencv.org/releases/) and [this more in-depth tutorial](https://opencv-java-tutorials.readthedocs.io/en/latest/01-installing-opencv-for-java.html) describe how to install it. Once it's installed, copy 
+### Building OpenCV <a name="building-opencv" id="building-openv"></a>
+
+The [official site](https://opencv.org/releases/) and [the somewhat dated tutorial](https://opencv-java-tutorials.readthedocs.io/en/latest/01-installing-opencv-for-java.html) describe how to install it. These are, we believe, sufficient in Windows and Linux environments.
+
+#### Building OpenCV for macOS
+
+Installation on macOS<sup>&reg;</sup> is a bit more complicated. The easiest way is to use [Homebrew](https://brew.sh). _Homebrew_ is a package manager for macOS.  It is also available for Linux.
+
+Once _Homebrew_ is installed, you can read the instructions in [this slightly dated tutorial]() for background information. However, recent changes in _Homebrew_ have made some changes necessary.
+
+Specfically, current versions of _Homebrew_ need a bit more information about the Java environment to build the Java code for OpenCV. If this information is not provided, the installation of OpenCV appears to succeed, but the Java implemenation is not available. No errors are apparent. 
+
+As a result, the following steps are necessary to install OpenCV with Java support on macOS. The following are expected to be run from the Terminal application.
+
+1. If not already installed, install _Homebrew_ (see the [installation instructions](https://brew.sh).) The _Homebrew_ command is `brew`, so you will see its use below.
+2. If not already installed, install _Ant_ (used by the build process).
+    *`brew install ant`.
+    * Set the ANT_HOME environment variable to the installation location.  Due to some issues with the current versions of the _Ant_ installation, you may need to to set ANT_HOME to `/usr/local/Cellar/ant/1.10.11/libexec` where _1.10.11_ is the version of _Ant_ installed.
+3. If not already installed, install the XCode command-line tools.
+    * `xcode-select install`
+4. Prepare to use `brew` to install OpenCV. To do this, you will configure the OpenCV installation to include the Java libraries.  Do do this, you will edit the `brew` _formula_ (the instructions `brew` uses to do the installation.
```

Comment:
```suggestion
4. Prepare to use `brew` to install OpenCV. To do this, you will configure the OpenCV installation to include the Java libraries.  This is done by editing the `brew` _formula_ (the instructions `brew` uses to do the installation.
```

Or something along those lines

Reply:
I think I've spent enough time in `brew` this week that "do do" was just to hard to avoid...